src/fabric/src/fabric2_fdb.erl

```diff
@@ -1364,3 +1371,21 @@ run_on_commit_fun(Tx) ->
             Fun(),
             ok
     end.
+
+start_span_fun(Fun, Options, ExtraTags) ->
+    {name, OperationName} = erlang:fun_info(Fun, name),
```

Comment:
I added a handy `fun_to_op` function in ctrace which makes it super easy to put spans in throughout the code when debugging/optimizing:

https://github.com/apache/couchdb/blob/03f3054e5987bc0c1927bc417c41f3a518391f6b/src/ctrace/src/ctrace.erl#L176-L181

We might also want to make it optional so that `ctrace:with_span` accepts a `fun` and auto creates the name as that saves on the fairly repetitious `OpName = ctrace:fun_to_op/1` .